lobsterpy/structuregraph/graph.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) lobsterpy development team
+# Distributed under the terms of a BSD 3-Clause "New" or "Revised" License
+
+"""
+This package provides the modules for generating graph objects using lobsterpy data
+"""
+
+from typing import Optional
+from pymatgen.core.structure import Structure
+from pymatgen.io.lobster.lobsterenv import LobsterNeighbors
+from pymatgen.io.lobster.outputs import Charge
+from lobsterpy.cohp.analyze import Analysis
+
+
+class LobsterGraph:
+    """
+    LobsterGraph class for bonding data from Lobster
+
+    Attributes:
+        sg: return structure_graph object
+    """
+
+    def __init__(
+        self,
+        path_to_poscar: str,
+        path_to_charge: str,
+        path_to_cohpcar: str,
+        path_to_icohplist: str,
+        path_to_madelung: str,
+        add_additional_data_sg=True,
+        path_to_icooplist: Optional[str] = None,
+        path_to_icobilist: Optional[str] = None,
+        which_bonds: str = "all",
+        cutoff_icohp: float = 0.10,
+        start: str = None,
+    ):
+        """
+        This is a class to get structure graph object with bonding information from lobsterpy
```

Comment:
Could you add more description here, how the bonds are actually determined?

Reply:
Done